CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```

Comment:
n_: please link to the CNCF code of conduct as here. https://github.com/openebs/cstor-operators/blob/master/CODE_OF_CONDUCT.md

Reply:
Done.